src/log.js

```diff
@@ -211,12 +195,12 @@ export class Log {
 
   /**
    * Creates an error object.
-   * @param {string|!Error} errorOrMessage
+   * @param {...*} var_args
    * @return {!Error}
    */
-  createError(errorOrMessage) {
-    const error = typeof errorOrMessage == 'string' ?
-        new Error(errorOrMessage) : errorOrMessage;
+  createError(var_args) {
+    const error = createErrorVargs.apply(null,
```

Comment:
oops, this is from a different pr. got here accidentally. removed. though probably should fix there instead...